hddm/models/hddm_info.py

```diff
@@ -100,23 +103,416 @@ class HDDM(HDDMBase):
     """
 
     def __init__(self, *args, **kwargs):
-        self.slice_widths = {'a':1, 't':0.01, 'a_std': 1, 't_std': 0.15, 'sz': 1.1, 'v': 1.5,
-                             'st': 0.1, 'sv': 3, 'z_trans': 0.2, 'z': 0.1,
-                             'p_outlier':1., 'v_std': 1, 'alpha': 1.5, 'pos_alpha': 1.5}
+        # self.slice_widths = {'a':1, 't':0.01, 'a_std': 1, 't_std': 0.15, 'sz': 1.1, 'v': 1.5,
```

Comment:
I think you should create a new class for this and not change `HDDInfo`.

Reply:
This would be the only point where I would disagree. Maybe serious refactoring could facilitate that, but without it, not changing HDDMInfo would rather lead to code duplication on the level of the various HDDMnn classes. This was changed also to accomodate HDDMrl (see 'alpha' and 'pos_alpha' additions to self.slice_widths for example). If you strongly disagree with this I will defer to you of course.